react/Header/NotificationsLink/NotificationsLink.js

```diff
@@ -0,0 +1,54 @@
+import React, { Fragment } from 'react';
+import BellIcon from '../../BellIcon/BellIcon';
+import styles from './NotificationsLink.less';
+import { getCookie } from './utils';
+import { AUTHENTICATED } from '../../private/authStatusTypes';
+
+class NotificationsLink extends React.Component {
+  state = {
+    isInExperiment: false
+  }
+
+  componentDidMount() {
+    const EXPERIMENT_ID = '58CC4E12-A4E0-4C0B-BD2A-681FD98FDC2E';
```

Comment:
These consts could be moved outside the component

Reply:
Agree the `experimentId` and the `url` can be hoisted out, but the `visitorId` should remain.